src/main/docs/guide/openApiViews/rapidoc.adoc

```diff
@@ -52,17 +52,17 @@ See https://mrin9.github.io/RapiDoc/api.html[RapiDoc Options] for a description.
 To expose the `rapidoc` views, you also must expose the generated `yaml`:
 
 .Exposing Swagger YAML And Rapidoc Views
-[configuration]
+[source,yaml]
```

Comment:
we want to use the `configuration` macro

Reply:
fixed